config/pretty-routes.php

```diff
@@ -88,4 +114,41 @@
      */
 
     'locale_force' => false,
+
+    /*
+     * Enable open link in new tab.
+     *
+     * Useful for manual testing or visual checks.
+     *
+     * This option related with "dummy_variable_prefix"
+     */
+
+    'show_path_link' => true,
+
+    /*
+     * Laravel stores variable names with curly bracktes, this option replaces "prefix" and "variable name".
+     * So we can prevent url encode when browsing and we can get readable urls
+     *
+     * For example
+     * before admin/line/lines/{line_id} => admin/line/lines/%7Bline_id%7D
+     * after: admin/line/lines/{line_id} => admin/line/lines/param_line_id
```

Comment:
This option useful working with Postman or manual testing. Also I want to copy, paste, double click and typing (parameter value). Otherwise I have to delete curly braces. If there are many parameters this is time waste.

One more thing, if you are using `Route::resource()` laravel generates own routes. Like this,
`admin/line/lines/{line}/number/{number}/status/{status}...`
and after formatted
`admin/line/lines/line/number/number/status/status...`

Yes I know we can change parameter names with `->parameters(...)` method but I am not working always my projects. If I am maintaining someone else's project, I may not be able to add this method to hundreds of routes. In this case I need prefix. This option only for route links, the list is not affected. 

Depends to `show_path_link``option. You can see open new tab icon.

![image](https://user-images.githubusercontent.com/17010054/106452693-48daba00-6499-11eb-9269-9b8c547877ca.png)

Reply:
Got it. On the pretty-routes page we will see `/foo/{bar}`, and when we open the link, we will see `/foo/param_bar`.

Okay.